README.md

```diff
@@ -109,11 +109,11 @@ These additional options are specific to pug templates and can be configured in
 
   Choices:
 
-  - `'always'` _default_ -> Always separate attributes with commas.  
+  - `'always'` _default_ → Always separate attributes with commas.  
```

Comment:
I'm a little bit afraid of these arrows ^^'
Do thy work on any OS and always will be rendered correctly?

Reply:
Good point. I think they're fine in 2020, but happy to switch back :)